ui/src/app/pages/admin/admin-workspace.tsx

```diff
@@ -227,13 +235,23 @@ class AdminWorkspaceImpl extends React.Component<UrlParamsProps, State> {
             )}
           </div>
           <h3>Collaborators</h3>
-          <div className='collaborators' style={{marginTop: '1rem'}}>
-            {collaborators.map((userRole, i) =>
-              <div key={i}>
-                {userRole.email + ': ' + userRole.role}
-              </div>
-            )}
-          </div>
+          <FlexRow className='collaborators' style={{marginTop: '1rem'}}>
+            <FlexColumn>
+              {collaborators.map((userRole, i) =>
+                <div key={i}>
+                  {userRole.email + ': ' + userRole.role}
+                </div>
+              )}
+            </FlexColumn>
+            <FlexColumn>
+              <Button style={{marginLeft: '0.5rem'}}
+                      onClick={() => {
+                        this.setState({loadingData: true});
+                        const writersAndOwners = fp.filter(userRole => userRole.role === WorkspaceAccessLevel.OWNER || userRole.role === WorkspaceAccessLevel.WRITER, collaborators);
```

Comment:
BTW, we don't yet have an approved way of doing this, but we should be able to tell you which user is actually running the cluster, so you don't have to knock out all the others.

Reply:
I know. This PR is meant to create a nuclear option. If the person running the cluster controls more than one user on the workspace, they'll just start the cluster with another user. Knocking out a single user should be done with the disable user script and/or the user admin.